src/psyclone/psyir/nodes/call.py

```diff
@@ -111,17 +129,19 @@ def create(cls, routine, arguments=()):
         :raises TypeError: if the routine argument is not a RoutineSymbol.
         :raises GenerationError: if the arguments argument is not an Iterable.
 
-        '''
+        """
         if not isinstance(routine, (Reference, RoutineSymbol)):
             raise TypeError(
                 f"The Call routine argument should be a Reference to a "
                 f"RoutineSymbol or a RoutineSymbol, but "
-                f"found '{type(routine).__name__}'.")
+                f"found '{type(routine).__name__}'."
```

Comment:
Please revert these formatting changes too. I don't mind if you're already changing that bit of code but otherwise it could well cause pain for other people.

Reply:
OK